evergreen/run-atlas-connectivity-tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# DO NOT set xtrace
+set -o errexit  # Exit the script with error if any of the commands fail
+
+############################################
+#            Main Program                  #
+############################################
+
+# Provision the correct connection string and set up SSL if needed
+for var in TMP TEMP NUGET_PACKAGES NUGET_HTTP_CACHE_PATH APPDATA; do setx $var z:\\data\\tmp; export $var=z:\\data\\tmp; done
```

Comment:
Not totally sure why these environment variables are needed but they were in run-tests.sh so I copied them here too.

They do make sense (various tools needing a temp directory).